lusidtools/pandas_utils/lusid_pandas.py

```diff
@@ -2,8 +2,7 @@
 from flatten_json import flatten
 
 
-def lusid_response_to_data_frame(lusid_response):
-
+def lusid_response_to_data_frame(lusid_response, format_df: bool = False):
```

Comment:
I think `format_df` should be rename to something like `rename_properties` to make it clear what its doing. It would still be useful at some point to have the `{str: Callable}` param to allow customisation of the columns

Reply:
Sure, thats renamed. 
I will create a JIRA for {str: callable} and add this to my to-do